src/web/src/components/Posts/Post.tsx

```diff
@@ -284,7 +297,7 @@ const PostComponent = ({ postUrl }: Props) => {
       <div className={classes.content}>
         <section
           ref={sectionEl}
-          className="telescope-post-content"
+          className={`telescope-post-content ${extractBlogClassName(new URL(post.url).hostname)}`}
```

Comment:
let's move the `new URL(post.url).hostname)` bit into `extractBlogClassName` and just pass the `post.url` here.

Reply:
Is something like this what you had in mind?
```js
const extractBlogClassName = (url: string) => {
  const blogClassName = new URL(url).hostname;
  if (blogClassName.endsWith('medium.com')) {
    return 'is-medium';
  } 
  // ... rest of function ...
}
```
This is what I pushed just now, I've checked and the class name decoration still works as intended